src/languageclient.rs

```diff
@@ -2074,6 +2074,50 @@ pub trait ILanguageClient: IVim {
         Ok(())
     }
 
+    fn window_handleProgress(&self, params: &Option<Params>) -> Result<()> {
+        info!("Begin {}", NOTIFICATION__WindowProgress);
+        let params: WindowProgressParams = serde_json::from_value(params.clone().to_value())?;
```

Comment:
`.to_value()` is redundant.

Reply:
That's how it's done elsewhere; without it it gives an error that it expects a `Value` not an `Option`.

Reply:
Ahh, I see. Apologies.